gitlabform/configuration/core.py

```diff
@@ -162,13 +163,64 @@ def is_group_skipped(self, group) -> bool:
     def is_skipped(self, an_array: list, item: str) -> bool:
         pass
 
+    @staticmethod
+    def validate_break_inheritance_flag(config, **kwargs):
```

Comment:
Nit: I think that we don't need `**kwargs` here, it's making code harder to read. Plain parameters perhaps with default values would be easier.

Reply:
I have refactored this function so it is a lot simpler to understand. After I explained what the code was supposed to do it made me realize, I didn't need the other parameters.